tempest/lib/javelin

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash -x
+#---------------------------------------------------------------
+# Project         : Javelin
+# File            : javelin
+# Copyright       : (C) 2015 by
+# Author          : Joe H. Rahme
+# Created On      : Jan 23 2015
+# Purpose         : Use Javelin for testing
+#---------------------------------------------------------------
+
+source /etc/config-tools/openrc.sh
+
+tempest_dir="/usr/share/openstack-tempest-juno"
+resourcefile="javelin-resource.yml"
+lastjavelin="/opt/javelin-latest-run"   # Make sure that this file survives upgrades by mentioning it in edeploy metadata
+
+javelin_get_or_create_resourcefile () {
+    if [ ! -r "$resourcefile" ]; then
+	cat > "$resourcefile" <<EOF
+tenants:
+  - javelin
+  - discuss
+
+users:
+  - name: javelin
+    pass: gungnir
+    tenant: javelin
+  - name: javelin2
+    pass: gungnir2
+    tenant: discuss
```

Comment:
(discussed offline) we also may want to test containers, volumes, instances and networks.